pkg/tempo/automaticloggingprocessor/automaticloggingprocessor.go

```diff
@@ -222,10 +248,11 @@ func (p *automaticLoggingProcessor) exportToLoki(kind string, traceID string, ke
 		return
 	}
 
+	// Add loki label
+	labels[model.LabelName(p.cfg.Overrides.LokiTag)] = model.LabelValue(kind)
```

Comment:
spanLabels can return nil, does this work?

Reply:
Changed to return a zero value, nice catch!